src/coreclr/src/jit/emitarm.cpp

```diff
@@ -3663,7 +3674,8 @@ void emitter::emitIns_R_S(instruction ins, emitAttr attr, regNumber reg1, int va
 }
 
 // generate the offset of &varx + offs into a register
-void emitter::emitIns_genStackOffset(regNumber r, int varx, int offs, bool isFloatUsage)
+// also capture the 'baseReg' used for generating movw/movt
```

Comment:
Could you add a standard function header here? And I would also rename this parameter as `pBaseReg` and describe it as "an out parameter to capture the 'baseReg' used ..."